crates/air_r_formatter/src/context.rs

```diff
@@ -138,20 +138,20 @@ impl RFormatOptions {
 }
 
 impl FormatOptions for RFormatOptions {
-    fn indent_style(&self) -> IndentStyle {
-        self.indent_style
+    fn indent_style(&self) -> biome_formatter::IndentStyle {
+        self.indent_style.into()
     }
 
-    fn indent_width(&self) -> IndentWidth {
-        self.indent_width
+    fn indent_width(&self) -> biome_formatter::IndentWidth {
+        self.indent_width.into()
     }
 
-    fn line_width(&self) -> LineWidth {
-        self.line_width
+    fn line_width(&self) -> biome_formatter::LineWidth {
+        self.line_width.into()
     }
 
-    fn line_ending(&self) -> LineEnding {
-        self.line_ending
+    fn line_ending(&self) -> biome_formatter::LineEnding {
+        self.line_ending.into()
```

Comment:
This is the only place we convert from our `settings::` types to `biome_formatter::` types. That's it! That's all the glue! Go traits!